config/app_config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda-proxy/metrics"
+	"github.com/urfave/cli/v2"
+)
+
+// AppConfig ... Highest order config. Stores all relevant fields necessary for running both proxy & metrics servers.
+type AppConfig struct {
+	EigenDAConfig ProxyConfig
+	MetricsCfg    metrics.Config
```

Comment:
```suggestion
	EigenDAConfig ProxyConfig
	MetricsConfig    metrics.Config
```

Reply:
consistency between both fields

Reply:
Wait also EigenDAConfig feels wrong. Its a ProxyConfig which contains configs for all internal services, not only eigenda backend.

Reply:
Also personally find it a bit weird that our highest-level app config is "proxy + metrics". Feel like it puts too much emphasis on the servers, whereas the proxy is a bunch of services + servers. I feel like MetrisConfig should just be moved to the ProxyConfig and ProxyConfig should be the highest level "app" config.

Reply:
I agree that there are a substantial number of changes that can be made here. Are you ok with addressing these comments in a [followup PR](https://linear.app/eigenlabs/issue/DAINT-327/cleanup-v2-proxy-config-code), to avoid  additional scope creep of this PR?